engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py

```diff
@@ -53,6 +55,7 @@ class VllmRbProperties(Properties):
     preloaded_model: Optional[Any] = None
     max_logprobs: Optional[int] = 20
     enable_chunked_prefill: Optional[bool] = False
+    cpu_offload_gb_per_gpu: Optional[int] = None
```

Comment:
Should this be `cpu_offload_gb`([ref](https://github.com/vllm-project/vllm/blob/e0c15758b85827dcac78379e60ea975ebc0ec795/vllm/engine/arg_utils.py#L57))?

Reply:
cpu_offload_gb is confusing... the actual meaning of it is cpu_offload_gb_per_gpu. If you set to 40 and do TP4, it will allocate 160GB CPU memory